### PR TITLE
Pin debian kernel

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This PR pins the kernel for the Debian distribution to version `linux-image-4.19.0-9-cloud-amd64`.

## 💭 Motivation and Context

I generated a Guacamole AMIs for [production](https://github.com/cisagov/guacamole-packer/releases/tag/v0.1.3) and [staging](https://github.com/cisagov/guacamole-packer/releases/tag/v0.1.3-rc.2) late last week.  @dav3r and I have been seeing those instances randomly lock up.  We believe that this is because they are Debian images running with the linux kernel package `linux-image-4.19.0-10-cloud-amd64` (`4.19.0-10-cloud-amd64 #1 SMP Debian 4.19.132-1 (2020-07-24) x86_64`), while the last known good AMIs were running `linux-image-4.19.0-9-cloud-amd64` (`4.19.0-9-cloud-amd64 #1 SMP Debian 4.19.118-2+deb10u1 (2020-06-07) x86_64`).

There are a number of online reports of a recent kernel change causing such behavior, e.g. [here](https://forum.proxmox.com/threads/kernel-5-4-44-causes-system-freeze-on-hp-microserver-gen8.72050/) and [here](https://lkml.org/lkml/2020/7/20/883).  The problematic change is [confirmed to be in the Debian `linux-image-4.19.0-10-cloud-amd64` package](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=966846).

I have manually set up env1 in staging and env4 in production to save a `kdump` if they encounter a kernel panic.  That should allow us to determine for certain whether the kernel issue referred to above is indeed the culprit the next time one of those instances locks up.

## 🧪 Testing

All molecule tests and pre-commit hooks pass.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tasks/Debian_10_hold_kernel.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    name:
+      - linux-headers-cloud-amd64
+      - linux-image-cloud-amd64
+    state: absent
+- name: Install specific version of kernel (Debian)
+  apt:
+    name:
+      - linux-headers-4.19.0-9-cloud-amd64
+      - linux-image-4.19.0-9-cloud-amd64
+- name: Hold the kernel so that it is not upgrades via apt-get upgrade
```
Comment (dav3r) [inline tasks/Debian_10_hold_kernel.yml]: Typo: `upgrades` --> `upgraded`

**Review by dav3r — APPROVED** [review]
Aside from a minor typo, this all looks solid.  🌽 📌

File: tasks/Debian_10_hold_kernel.yml (hunk 1 above)
Comment (jsf9k) [reply tasks/Debian_10_hold_kernel.yml]: See commit 5a4a677.

**Review by mcdonnnj — APPROVED** [review]
Makes sense, nice detective work 🕵️‍♂️

**Pull request merged**